client/updateaccess.go

```diff
@@ -49,6 +53,26 @@ var updateAccessService = cmdUpdateAccess.Flag.Bool("S", false, "")
 var updateAccessServicePrefix = cmdUpdateAccess.Flag.Bool("N", false, "")
 
 func runUpdateAccess(cmd *Command, args []string) {
+	if *updateAccessACL != "" {
+		if len(args) != 1 {
+			fatalf("access takes exactly one arguments. See 'knox help access'")
```

Comment:
This sounds wrong to me, can't it also take more than one arg based on situation?

Reply:
it can take more than one arg if you *don't* specify the --acl flag (which is the check on line 56)

Reply:
I fixed the wording to say it takes one arg when --acl is used